app/scripts/modules/core/cluster/categorySelection/categorySelection.service.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+
+let angular = require('angular');
+
+module.exports = angular.module('spinnaker.categorySelection.service', [
+  require('../../account/account.service.js'),
+  require('../../config/settings.js'),
+  require('../../utils/lodash.js'),
+])
+  .factory('categorySelectionService', function($uibModal, $q, _, accountService, settings) {
+    function availableCategories() {
+      var available = [
+        {
+          name: 'serverGroup',
+          display: 'Server Group',
+        },
+      ];
+
+      if (settings.feature.jobs) {
+        available = available.concat([
```

Comment:
`available = available.concat([{...}]` could be rewritten as `available.push({name: 'job', display: 'Job'})`